media-fonts/ttf-ancient-fonts/ttf-ancient-fonts-9.17.ebuild

```diff
@@ -0,0 +1,20 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+MY_PV="2.60"
```

Comment:
What's the reason for PV=9.17? This package includes multiple fonts, and only some of them are version 9.17. PV=9.17 doesn’t seem accurate to me.

Reply:
Because the author has numbered his work/license (2020 link footer) for all fonts with symbola version. It is the result of my archaeological research:-)
[2020 = 13](https://web.archive.org/web/20200330173723/https://dn-works.com/ufas/)
[ 2019 = 12](https://web.archive.org/web/20191112125115/http://users.teilar.gr:80/~g1951d)
[2018 = 11](https://web.archive.org/web/20180630020732/http://users.teilar.gr/~g1951d/)
[2018 = 10](https://web.archive.org/web/20180119031702/http://users.teilar.gr/~g1951d/)
[2017 = 9 this package](https://web.archive.org/web/20171031034440/http://users.teilar.gr:80/~g1951d)

If you don't agree, I wil use 2.60

Reply:
I prefer 2.60 simply because everyone using Debian source is doing it, but I guess it's up to you.
https://repology.org/project/ttf-ancient-fonts/versions

Reply:
Versioning should be coming from upstream, but it's singular here. I'll change version to 2.60 soon. 
Thanks for review !